apps/mobile/src/libs/components/background-wrapper/background-wrapper.tsx

```diff
@@ -0,0 +1,52 @@
+import { ReactNode } from "react";
+
+import { Image, View } from "~/libs/components/components";
+import { globalStyles } from "~/libs/styles/styles";
+import { type ImageSourcePropType } from "~/libs/types/types";
+
+import { Planet } from "./libs/components/planet/planet";
+import { styles } from "./styles";
+
+type Properties = {
+	children: ReactNode;
+};
+
+const BackgroundWrapper = ({ children }: Properties) => {
+	return (
+		<View style={[globalStyles.flex1, styles.container]}>
+			<Image
+				source={
+					require("~/assets/images/top-circle.png") as ImageSourcePropType
+				}
+				style={[styles.image, styles.topImage]}
+			/>
+			<View style={[styles.dot, styles.dotPositionTop]} />
+			<View style={[styles.dot, styles.dotPositionRight]} />
+			<View style={[styles.dot, styles.dotPositionRightBottom]} />
+			<View style={[styles.dot, styles.dotPositionBottom]} />
+			<View style={[styles.dot, styles.dotPositionLeft]} />
+			<Planet color={"green"} size={"sm"} style={styles.topPlanet} />
```

Comment:
```suggestion
			<Planet color="green" size={"sm"} style={styles.topPlanet} />
```
I believe it's redundant, same for all